Version 1 - User Guide for WRD by Dancho and Desi

dessyordanova opened the pull request.

dessyordanova commented:
@ighristov, the WebServiceDataSource is adjusted to use the https://demos.telerik.com/reporting/api/data/productsales. Hence, the Getting Started gif video and respective screenshots are recaptured. 
Next steps:
I believe that we can proceed with uploading the videos to the Telerik account if @ighristov or @elkinmilen do not have any other comments. @yordan-mitev, could you please assist with the video upload task?

elkinmilen reviewed (dismissed):
You have done tremendous amount of work with a lot of diligence and attention to detail. Thank you! It is a solid start of our end-users onboarding experience. Videos are easy to consume and follow.

I have left inline comments - some smaller, some a bit more substantial. Please consider each one and apply up to your preference.
Bigger considerations:
- I miss the Assets Manager concept and how to use it - To organize your reports and the rest of the resources , and then to consume the assets, for example, when setting a Value for PictureBox.
- I miss the WRD Preferences explanation, no matter these are kind of trivial, it might be helpful to know for example that drag-drop to move and resize sticks to a grid (and the grid size is determined in a Report property), and the items snap to each other and these can be controlled.
- (this is also present as inline comment) I miss the Shared Data Sources concept. It is quite powerful concept allowing business org to separate responsibilities of data retrieval setup and data consumption in visual reports.

Please apply the smaller comments and merge. I click Approve. The rest should be a separate PR. When ready, please ask for review.

dessyordanova commented on user-guide/components/data-sources.md:
```diff
@@ -0,0 +1,249 @@
+
+</style>
+
+# Data Sources
```
We will mention the concept in a few sentences here now. A separate article will be planned for Version 2.

dessyordanova commented:
@elkinmilen, thank you for the provided feedback. It was all relevant and correct, so I addressed all of your comments. I am sharing the draft plan for the next version:
Next Steps: 
- Q4 2025: gather and analyze data and KPIs of the User Guide docs 
- Version 2: topics to cover (separate articles): 
1. Main menu 
2. Assets Manager: concept and how to use it - To organize your reports and the rest of the resources , and then to consume the assets, for example, when setting a Value for PictureBox.
3. WRD Preferences explanation, no matter these are kind of trivial, it might be helpful to know for example that drag-drop to move and resize sticks to a grid (and the grid size is determined in a Report property), and the items snap to each other and these can be controlled.
4. Creating Master-Detail Reports with SubReports: add article / section of the alternative approach to the parameters binding: using the DataSource property of the SubReport item. It is only applicable, if the detail report definition is data bound report (not data bound table within it), but allows binding hierarchical data source to master and detail reports in an understandable way.
5. SharedDataSource: The SharedDataSource component enables report designers to easily create and reuse a data source component across multiple reports.

ighristov reviewed (approved):
Thanks again for all the efforts!
